meta/FlexibleSUSY.m

```diff
@@ -2595,7 +2595,7 @@ corresponding tadpole is real or imaginary (only in models with CP
 
       WriteOut`ReplaceInFiles[files,
         {"@AMMZBosonField@"       -> CXXDiagrams`CXXNameOfField[TreeMasses`GetZBoson[]],
-         "@AMMCalculation@"       -> TextFormatting`IndentText[calculation],
+         "@AMMCalculation@"       -> TextFormatting`IndentText[TextFormatting`IndentText[calculation]],
```

Comment:
I was thinking whether we should start using
```mathematica
Nest[TextFormatting`IndentText, calculation, 2]
```
instead of
```mathematica
TextFormatting`IndentText[TextFormatting`IndentText[calculation]]
```
It's shorter and in complicated cases it becomes difficult to figure out at first glance how many levels deep are we.

Reply:
Good point!

Reply:
fixed in b5926a3ab